src/structures/APIMessage.js

```diff
@@ -182,19 +169,19 @@ class APIMessage {
       typeof this.options.allowedMentions === 'undefined'
         ? this.target.client.options.allowedMentions
         : this.options.allowedMentions;
-    if (this.options.reply) {
-      const id = this.target.client.users.resolveID(this.options.reply);
-      if (allowedMentions) {
-        // Clone the object as not to alter the ClientOptions object
-        allowedMentions = Util.cloneObject(allowedMentions);
-        const parsed = allowedMentions.parse && allowedMentions.parse.includes('users');
-        // Check if the mention won't be parsed, and isn't supplied in `users`
-        if (!parsed && !(allowedMentions.users && allowedMentions.users.includes(id))) {
-          if (!allowedMentions.users) allowedMentions.users = [];
-          allowedMentions.users.push(id);
-        }
-      } else {
-        allowedMentions = { users: [id] };
+
+    if (allowedMentions) {
+      allowedMentions.replied_user = allowedMentions.repliedUser;
+      delete allowedMentions.repliedUser;
```

Comment:
this would modify the original `allowedMentions` object in the `ClientOptions`, or the object given in the `MessageOptions`, perhaps implement the same `allowedMentions = Util.cloneObject(allowedMentions)` behaviour as before if `allowedMentions && typeof allowedMentions.repliedUser === 'boolean'`

Reply:
Is this resolved now? If so, please mark it as such

Reply:
I believe so, from fbb1c93.